.gitignore

```diff
@@ -62,3 +62,5 @@ typings/
 
 # vscode configuration
 .vscode
+
+optimized.js
```

Comment:
Wouldn't it be nicer to generate this file in some more generic cache/build/dist folder and ignore that instead?